### PR TITLE
Add parameter judgment for commands that do not require parameters

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: cmd/buildah/containers.go [hunk 1]
```diff
@@ -90,6 +90,9 @@ var (
 )
 
 func containersCmd(c *cli.Context) error {
+	if len(c.Args()) > 0 {
+		return errors.New("'buildah containers' accepts no arguments")
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/containers.go]: I think I'd prefer "does not accept arguments" over "accepts no arguments" for both of these errors.

Comment (TomSweeneyRedHat) [reply cmd/buildah/containers.go]: I'm sitting on the fence about the need for this change or just changing it to a warning message and then just continue on.  @rhatdan WDYT?

Comment (rhatdan) [reply cmd/buildah/containers.go]: I prefer Tom's message.  I am fine with this error-ing out.

Comment (q384566678) [reply cmd/buildah/containers.go]: updated, PTAL.

**Comment by q384566678** [comment]
>Code LGTM but I am missing tests

Do we still need to perform a special test on this? Each command has a judgment on the number of input parameters. In this case, the parameters need to be tested for each command. I don't think it is necessary.

**Comment by TomSweeneyRedHat** [comment]
LGTM.  @ripculd I think the tests have been provided in another PR.  Are you OK with this PR?

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**